templates/CRM/Financial/Page/BatchTransaction.tpl

```diff
@@ -75,17 +75,17 @@ CRM.$(function($) {
 });
 function assignRemove(recordID, op) {
   var recordBAO = 'CRM_Batch_BAO_Batch';
-  if (op == 'assign' || op == 'remove') {
-    recordBAO = 'CRM_Batch_BAO_EntityBatch';   
+  if (op === 'assign' || op === 'remove') {
```

Comment:
@eileenmcnaughton does === do the same thing in JS as in php?

Reply:
@seamuslee001 I think so since it was my IDE that wanted this change

Reply:
@seamuslee001 seems to be working fine 
![image](https://user-images.githubusercontent.com/336308/148705142-26ff5f63-fe95-4196-8ea1-ba337884df01.png)

Reply:
Yes it does